types/part_set.go

```diff
@@ -133,7 +133,7 @@ func (psh *PartSetHeader) ToProto() cmtproto.PartSetHeader {
 	}
 }
 
-// FromProto sets a protobuf PartSetHeader to the given pointer
+// PartSetHeaderFromProto sets a protobuf PartSetHeader to the given pointer
```

Comment:
[nit] I'm totally on-board with these comment fixes but it may make sense to make them upstream to minimize celestia-core's diff with CometBFT.

Otherwise, they may accidentally get overwritten next time we pull from upstream.

Reply:
That is a good idea, will do that.

Reply:
I'll open a separate PR to upstream the go documentation fixes.